bin/es_cli.py

```diff
@@ -110,52 +104,57 @@ def cmd_list_crashids(ctx, index):
 @click.argument("index", nargs=1)
 @click.pass_context
 def cmd_print_mapping(ctx, index):
-    context = get_conn()
-    doctype = context.get_doctype()
-    conn = context.connection()
-    indices_client = IndicesClient(conn)
-    resp = indices_client.get_mapping(index=index)
-    mapping = resp[index]["mappings"][doctype]["properties"]
-    click.echo(json.dumps(mapping, indent=2, sort_keys=True))
+    crashstorage = get_crashstorage()
+    doctype = crashstorage.get_doctype()
+    with crashstorage.client() as conn:
```

Comment:
I'm pretty sure I'm being inconsistent with naming things. Crashstorage instances should have a connection which is a ConnectionContext instance and those should have a client which is the client from the library it's using to access the storage system (e.g. boto, elasticsearch-py, etc).

I'll clean up consistency issues in another PR.